hbase-server/src/main/java/org/apache/hadoop/hbase/namequeues/impl/SlowLogQueueService.java

```diff
@@ -64,10 +64,16 @@ public class SlowLogQueueService implements NamedQueueService {
   private final boolean isSlowLogTableEnabled;
   private final SlowLogPersistentService slowLogPersistentService;
   private final Queue<TooSlowLog.SlowLogPayload> slowLogQueue;
+  private final int slowLogOperationJsonMaxCols;
+  private final boolean slowLogOperationJsonEnabled;
 
   public SlowLogQueueService(Configuration conf) {
     this.isOnlineLogProviderEnabled = conf.getBoolean(HConstants.SLOW_LOG_BUFFER_ENABLED_KEY,
       HConstants.DEFAULT_ONLINE_LOG_PROVIDER_ENABLED);
+    this.slowLogOperationJsonMaxCols = conf.getInt(HConstants.SLOW_LOG_OPERATION_JSON_MAX_COLS,
+      HConstants.SLOW_LOG_OPERATION_JSON_MAX_COLS_DEFAULT);
+    this.slowLogOperationJsonEnabled = conf.getBoolean(HConstants.SLOW_LOG_OPERATION_JSON_ENABLED,
+      HConstants.SLOW_LOG_OPERATION_JSON_ENABLED_DEFAULT);
```

Comment:
We could consider registering this class with the `ConfigurationManager` and implementing `ConfigurationObserver` to make these dynamic, but I omitted that complication from this initial draft. I'm open to input on whether that should make it into our first upstream iteration.